### ISSUE TITLE
Checkpoint #3

### ISSUE THREAD

**Issue by dmueller3** [dmueller3/FantasyFootballScheduler#5]
Hi @pawaitemadisoncollege

My checkpoint #3 is ready for review! To follow the mvp, click 'generate a schedule', then follow the form inputs. The database stuff will be shown in the team name dropdowns!

AWS link here: http://fantasy-fb-scheduler-env.eba-mrvcct3j.us-east-2.elasticbeanstalk.com/

Here's an image of what is looks like:
![image](https://github.com/user-attachments/assets/992fa819-fb84-44ca-9dc2-40706d3fe470)

**Comment by pawaitemadisoncollege** [comment]
Hi @dmueller3!  Congrats on completing checkpoint 3! 

**Deployed to AWS with auth implemented** - I was able to sign up and log in.

**at least one JSP that displays data from the database is implemented** - I was able to see data in the teams dropdowns as you showed in your screenshot. 

**add your deployed app url to the indie project list**

- [x] Add your deployed app url here https://github.com/mad-ent-java-f24/student/blob/main/IndividualProjects.md

**update your time log and project plan**

- [x] Are these up-to-date? Github shows they were last updated last month.

**Complete/respond to the two items above to demonstrate a met level for checkpoint 3.**

**Other**

I took a bit of time to look through your project code and things are looking pretty good at this point! With 3 weeks to go, I think you are in good shape for hitting the mark!   Some next steps might include: 

1. Use jstl import tags rather than the directive: https://github.com/dmueller3/FantasyFootballScheduler/blob/3d28ff920ad7716be3cf5a206c0dd8b178d99738/src/main/webapp/scheduleResult.jsp#L3 and other pages. In some cases you are using the import tag: https://github.com/dmueller3/FantasyFootballScheduler/blob/3d28ff920ad7716be3cf5a206c0dd8b178d99738/src/main/webapp/generationSettings.jsp#L8. Not sure if there was a reason to do it both ways? 
2. Add your user entity, and unit test its connection with the database.
3. Implement the generic dao
4. Add logic to add the user to the db if they are not already there (part of auth)

Looking forward to seeing more functionality soon. Let me know if you'd like another review in the week or so!
